runtime/integration-tests/src/utils/extrinsics.rs

```diff
@@ -151,8 +151,6 @@ fn signed_extra_relay(nonce: RelayIndex) -> RelaySignedExtra {
 		frame_system::CheckNonce::<RelayRuntime>::from(nonce),
 		frame_system::CheckWeight::<RelayRuntime>::new(),
 		pallet_transaction_payment::ChargeTransactionPayment::<RelayRuntime>::from(0),
-		#[cfg(not(feature = "runtime-development"))]
-		polkadot_runtime_common::claims::PrevalidateAttests::<RelayRuntime>::new(),
```

Comment:
Why the removal?

Reply:
Nongeneric tests are always run in CI under `runtime-development,` so this implies that that line is never compiled. In the case of other runtimes, we have now the generic part which should take care of that. Actually, we no longer have `runtime-x` features.

Please correct me if I'm missing something here, but CI will always tests against the development.